lib/api/attachUrl.js

```diff
@@ -25,6 +25,14 @@ async function serve (req, res) {
     // error
   }
 
+  if (typeString === 'Pick an attachment type...') {
+    if (!req.accepts('text/html')) {
+      res.status(400).send('Must pick an attachment type')
+    } else {
+      res.redirect(url)
```

Comment:
Shouldn't it display an error on the page?

Reply:
Agreed, but I was not sure how to do that.  Ideally, there should be popups with this one and the add/edit/remove fields ones when something goes wrong.  I settled on simply refreshing the page.